### PR TITLE
Add date range to map

### LINKED ISSUE #96: [Frontend] Put date range of data on map page

### PR DESCRIPTION
**Pull request by dcjohnson24**
## Checklist 
- [ ] I am requesting to merge into the dev branch 
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

| Before      | After       |
| ----------- | ----------- |
     
 before-map 
 
map-dates-friendly

after-map

Fixes #96

### CODE REVIEW

File: src/Pages/InteractiveMap.js
[the remark this answers is not in the document]
Comment (dcjohnson24) [reply src/Pages/InteractiveMap.js:1]: Thanks, I've changed the variable names to camelCase in `InteractiveMap.js` and in `data.json`.

**Comment by hkier** [comment]
The suggest implementation is not user friendly.  We have to remember the people using this site may or may not be technologically or scientific literate.  The date needs to be in a standard US English readable format and not just copied character by character from the data.json file.

2022-05-23 should be in either of these standard formats,

May 5, 2022
5/23/2022

**Comment by dcjohnson24** [comment]
Thanks, I'll convert it to one of those two suggestions.

**Comment by hkier** [comment]
I think I have integrated this code into my conflicting PR.  However, I do not have the data file in the right format.  Therefore, I am not able to test the change in my code.

**Comment by lauriemerrell** [comment]
This feature got added elsewhere

**Pull request closed without merging**